packages/client/src/intros/shared/Intro.tsx

```diff
@@ -43,14 +45,18 @@ export default ({
 
   return (
     <>
-      <Paragraph gutterBottom={usableForBullets.length > 0 ? 8 : undefined}>
-        {t("introPage.common.check for permit intro")}{" "}
-        {usableForBullets.length > 0 && t("introPage.common.permit for")}
+      { needReport && <Paragraph>
+        {t("introPage.demolition.need report")}{" "}
```

Comment:
I would change `needReport` to `demolitionChecker` (or something) because that's where the text is from.
`needReport` > `demolition.need report` looks off

Reply:
This is a generic intro component. Should be reusable and contain no logic for specific checkers. Translations could come from common if really needed. I think needReport is also not the best name. `outcomesIncludeNeedReport` would be better. But the text cannot contain demolition text. Another solution would be to check if the current checker has a key in the translation file, and if so, render the text here.

Reply:
Please add new props to this component to `Intro.test.tsx`